lib/i18n/tasks/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module I18n
   module Tasks
-    VERSION = '0.9.30'
+    VERSION = '0.10.1'
```

Comment:
I wasn't sure what version scheme made sense for this level of change and project. Happy to set to whatever is needed.

Reply:
I'll bump the version in a separate commit (will be 0.9.31 instead because most users have the version locked to `~> 0.9.x`)

Reply:
reverted my version change
https://github.com/KicksiteDev/i18n-tasks/commit/51efc00b555409ed2447edcc7d078597ff24967d